packages/hardhat-helpers/src/networks.js

```diff
@@ -55,8 +55,10 @@ Object.keys(networks).forEach((key) => {
       name: networks[key].name,
       url: networks[key].provider,
       accounts: getAccounts(networks[key].name),
+      zksync: key === 'zkync',
```

Comment:
make sure that flag is set following recos here https://era.zksync.io/docs/tools/hardhat/migrating-to-zksync.html#configuration-changes